apps/web/src/pages/activities/components/ActivityItem.tsx

```diff
@@ -1,106 +1,136 @@
-              <Grid.Col span={10}>
-                <h3
-                  style={{
-                    margin: '0px',
-                    marginBottom: '8px',
-                  }}
-                  data-test-id="row-template-name"
-                >
-                  {item.template.name}
-                </h3>
-                <When truthy={status === ExecutionDetailsStatusEnum.SUCCESS}>
-                  <Text data-test-id="status-badge-item" color={colors.success}>
-                    Done
-                  </Text>
-                </When>
-                <When truthy={status === ExecutionDetailsStatusEnum.FAILED}>
-                  <Text data-test-id="status-badge-item" color={colors.error}>
-                    Failed
-                  </Text>
-                </When>
-              </Grid.Col>
-            </Grid>
-          </div>
-          <div>
-            <small>
-              <b>Date:</b> {format(new Date(item.createdAt), 'dd/MM/yyyy HH:mm')}
-            </small>
-            <div data-test-id="subscriber-id">
+                </Grid.Col>
+              </Grid>
+            </div>
+            <div>
               <small>
-                <b>Subscriber id:</b> {item.subscriber.id}
+                <b>Date:</b> {format(new Date(item.createdAt), 'dd/MM/yyyy HH:mm')}
               </small>
+              <div data-test-id="subscriber-id">
+                <small>
+                  <b>Subscriber id:</b> {item.subscriber.id}
+                </small>
+              </div>
             </div>
-          </div>
-        </Grid.Col>
-        <Grid.Col span={9}>
-          <Grid
-            columns={item.jobs.length > 3 ? 10 : 12}
-            justify="end"
-            align="center"
-            sx={{
-              margin: 0,
-              height: '100%',
-            }}
-          >
-            {item.jobs.slice(0, 3).map((job) => (
-              <ActivityStep key={`activity-step-${job._id}`} span={item.jobs.length > 3 ? 3 : 4} job={job} />
-            ))}
-            <When truthy={item.jobs.length > 3}>
-              <Grid.Col span={1}>
-                <Text align="center" size="xl">
-                  +{item.jobs.length - 3}
-                </Text>
-              </Grid.Col>
-            </When>
-          </Grid>
-        </Grid.Col>
-      </Grid>
-    </ListItem>
+          </Grid.Col>
+          <Grid.Col span={9}>
+            <Grid
+              columns={item.jobs.length > 3 ? 10 : 12}
+              justify="end"
+              align="center"
+              sx={{
+                margin: 0,
+                height: '100%',
+              }}
+            >
+              {item.jobs
+                .filter((job) => job.type !== StepTypeEnum.TRIGGER)
+                .slice(0, 3)
+                .map((job) => (
+                  <ActivityStep
+                    isOld={isOld}
+                    key={`activity-step-${job._id}`}
+                    span={item.jobs.length > 3 ? 3 : 4}
```

Comment:
We do a lot this check so to avoid potential typos maybe we could convert it into a function to handle it in a single place.
```
const JOB_LENGTH_UPPER_THRESHOLD = 3;
const checkJobsLength = (item) => item.jobs.length > JOB_LENGTH_UPPER_THRESHOLD;
```